src/fu-common-version.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2017-2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#ifndef __FU_COMMON_VERSION_H__
+#define __FU_COMMON_VERSION_H__
+
+#include <gio/gio.h>
+
+/**
+ * FuVersionParseFlag:
+ * @FU_VERSION_PARSE_FLAG_NONE:			No flags set
+ * @FU_VERSION_PARSE_FLAG_USE_TRIPLET:		Use Microsoft-style version numbers
+ * @FU_VERSION_PARSE_FLAG_USE_BCD:		Use binary coded decimal notation
+ *
+ * The flags used when parsing version numbers.
+ **/
+typedef enum {
+	FU_VERSION_PARSE_FLAG_NONE		= 0,
+	FU_VERSION_PARSE_FLAG_USE_TRIPLET	= 1 << 0,
+	FU_VERSION_PARSE_FLAG_USE_BCD		= 1 << 1,	/* Since: 0.7.3 */
```

Comment:
Not technically true anymore that it came from 0.7.3

Reply:
Fixed, thanks.